src/super_gradients/training/datasets/detection_datasets/yolo_format_detection.py

```diff
@@ -184,6 +184,7 @@ def _load_annotation(self, sample_id: int) -> dict:
 
         yolo_format_target, invalid_labels = self._parse_yolo_label_file(
             label_file_path=label_path,
+            num_classes=len(self.all_classes_list),
```

Comment:
Here we assume all_classes_list is dense and sequential (E.g there are no gaps: 0, 2, 3, 4).
Is it true for all use cases? Even when we specify class_inclusion_list?

Reply:
`self.all_classes_list` is the dense list of classes.
If you set `class_inclusion_list`, it will not affect `self.all_classes_list` (it will affect `self.classes` instead)